configs/native.yaml

```diff
@@ -11,6 +11,7 @@ plugins:
   # - path: zed/
 #data: zip+http://robotics.ethz.ch/~asl-datasets/ijrr_euroc_mav_dataset/vicon_room1/V1_02_medium/V1_02_medium.zip?path=mav0
 data: ../../ILLIXR/data1/
+demo_data: ../../ILLIXR/demo_data/
```

Comment:
Please add to `native-ground-truth.yaml` as well.

Reply:
Done